packages/blocks/src/page-block/default/selection-manager.ts

```diff
@@ -256,6 +256,8 @@ export class DefaultSelectionManager {
     this.state.resetStartRange(e);
     this.state.refreshRichTextBoundsCache(this._mouseRoot);
     resetNativeSelection(null);
+    // deactivate quill keyboard event handler
+    (document.activeElement as HTMLTextAreaElement).blur();
```

Comment:
I think the Quill instance is an `HTMLDivElement`.

Reply:
How about this

```typescript
if (activeElement && 'blur' in activeElement && typeof activeElement['blur'] === 'function') {
  // deactivate quill keyboard event handler
    activeElement.blur();
  }
```

Reply:
This looks too verbose. `as HTMLDivElement` may be semantically better, otherwise, it may indicate that the instance here is the page title.

Reply:
OK, Get that!